classes/Server.py

```diff
@@ -9,6 +9,9 @@
 from .network.PacketType import PacketType
 from .player.Player import PlayerManager
 from .utils.Scheduler import SchedulerManager
+from classes.utils.Config import ConfigParser
+
+config = ConfigParser.load_config(1)
```

Comment:
Honestly, this is the wrong place to load it

Reply:
where should we load it instead?

Reply:
Probably within the McPy.py file as that is where I would expect it to be loaded

Reply:
the `NetworkController.start_process()` called in the `start` function here requires the host and port arguements, so that is why I load the config here.

Reply:
im going to try add host and port as arguments in the `start` function to not load the config here.